webroot/plots/plotly/io500-mdtest-easy-stat.html

```diff
@@ -15,9 +15,9 @@
+<script type="application/htmlwidget-sizing" data-for="htmlwidget-03b4d44919eff696b896">{"viewer":{"width":1000,"height":300,"padding":"0","fill":false},"browser":{"width":1000,"height":300,"padding":"0","fill":false}}</script>
```

Comment:
Note that having a single huge line here makes it basically impossible to review what is being changed in GitHub.   It would be much easier to read and review changes if it were split into separate lines (one per row).

Reply:
I don't like that we have to commit the graphs instead of having them generated (automatically) and then exclude the auto generated file.